app/templates/events/event_list.html

```diff
@@ -30,23 +30,22 @@ <h1 class="text-center mt-5">Events List for {{selectedTerm.description}}</h1>
     </div>
   </div>
 </div>
-<div>
+<div class="row">
   <ul class="nav nav-tabs nav-fill mx-3 mb-3" id="pills-tab" role="tablist">
-  <li class="nav-item" role="presentation">
+  <li class="col-lg-3 col-sm-6 col-12 nav-item" role="presentation">
```

Comment:
I think we should only break from 3 or 4 tabs per row to 1. This would be my suggested class list `text-nowrap col-12 col-md-3`. And I suppose we should decide if that should be `col-md-4` if we only are showing 3 tabs (no Bonners)